components/Questionnaire/Notification/index.vue

```diff
@@ -140,10 +169,12 @@ export default {
       }
     },
     showAgricultureForm () {
-      // @todo: create function for redirect link to form in new tab
+      // @todo: replace link google with actual link form
+      window.open('https://www.google.com')
```

Comment:
Do we agree to put the link on .env so that any update wont change the code?
put it on publicRuntimeConfig so that it can be called in any file #cmiiw

Reply:
Yes, we agree about that. I just did not add this setup in this PR. But, I will refactor it. Thanks for reminder me.